config/initializers/assets.rb

```diff
@@ -6,4 +6,8 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += %w( logo/logo-white.svg )
+# Rails.application.config.assets.precompile += %w( logo/logo-white.svg )
```

Comment:
[nitpick] I guess this could be removed completely

Reply:
Indeed. It is gone now.